src/controller/python/matter/commissioning/commissioning_flow_blocks.py

```diff
@@ -48,10 +49,14 @@ async def operational_credentials_commissioning(self, parameter: commissioning.P
             (commissioning.ROOT_ENDPOINT_ID, Clusters.BasicInformation.Attributes.ProductID)], returnClusterObject=True))[commissioning.ROOT_ENDPOINT_ID][Clusters.BasicInformation]
 
         self._logger.info("Getting AttestationNonce")
-        attestation_nonce = await self._credential_provider.get_attestation_nonce()
+        attestation_nonce: bytes = b''
+        if hasattr(self._credential_provider, "get_attestation_nonce") and callable(self._credential_provider.get_attestation_nonce):
```

Comment:
This seems like an odd complaint from mypy since this gets set in the init. Do you know why it thinks this might not be set?